shims/common/src/main/scala/org/apache/gluten/expression/ExpressionNames.scala

```diff
@@ -81,6 +81,8 @@ object ExpressionNames {
   final val NOT = "not"
   final val IS_NAN = "isnan"
   final val NANVL = "nanvl"
+  final val TRY_EVAL = "try"
+  final val CHECK_ADD = "plus"
```

Comment:
I note this is for Spark's `TryAdd`. And considering "plus" is velox-specific, it may be better to change it to:

`final val TRY_ADD = "try_add"`

Then, mapping it to velox function name in `SubstraitParser.cc`